Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2471,10 +2469,23 @@ extension BrowserViewController: TabManagerDelegate {
     addTabMenuChildren.append(openNewTab)
 
     var bookmarkMenuChildren: [UIAction] = []
+    
+    let containsWebPage = tabManager.selectedTab?.containsWebPage == true
+
+    if tabManager.openedWebsitesCount > 0, containsWebPage {
+      let bookmarkActiveTab = UIAction(
+        title: Strings.addToMenuItem,
+        image: UIImage(systemName: "book"),
+        handler: UIAction.deferredActionHandler { [unowned self] _ in
+          self.openAddBookmark()
+        })
 
-    if tabManager.openedWebsitesCount > 0 {
+      bookmarkMenuChildren.append(bookmarkActiveTab)
+    }
+    
+    if tabManager.tabsForCurrentMode.count > 1 {
       let bookmarkAllTabs = UIAction(
-        title: Strings.bookmarkAllTabsTitle,
+        title: String(format: Strings.bookmarkAllTabsTitle, tabManager.tabsForCurrentMode.count),
```

Comment:
Doesn't matter for this string in particular, but you should use `String.localizedStringWithFormat(...)` instead for localized strings as they take into account formatting order if the translator adds them

Reply:
Fixed at https://github.com/brave/brave-ios/pull/5236/commits/013a9ecefd1849141e901f1d9fb053db9ca938cf